### DISCUSSION TITLE
Email Capabilites

### DISCUSSION THREAD

**Discussion by bglatzer** [blueseerERP/blueseer#99] [Q&A] [answered]
Hi, 
Does BlueSeer have email capabilities? Like to send and receive emails to customers, vendors, etc.?
Thanks

**Comment by vaughnte** [comment, marked as answer]
It cannot receive emails, but it can send emails for certain modules.   For example, you can send an email for an invoice to the customer if the customer email is set up ....and the mail server is define/config'd in System Control.    Another email scenario would be signoff's within the organization for Requisitions (under the purchasing menu) and Engineering change control sign offs.
